spring-integration-redis/src/test/java/org/springframework/integration/redis/outbound/RedisOutboundGatewayTests.java

```diff
@@ -161,7 +161,7 @@ public void testMGetCommand() {
 		byte[] value2 = "bar2".getBytes();
 		connection.set("foo1".getBytes(), value1);
 		connection.set("foo2".getBytes(), value2);
-		this.mgetCommandChannel.send(MessageBuilder.withPayload(new String [] {"foo1", "foo2"}).build());
+		this.mgetCommandChannel.send(MessageBuilder.withPayload(new String[] {"foo1", "foo2"}).build());
```

Comment:
Yes, I did that in my IDEA as well to meet the same code formatting as your.
BTW, I have just merged it, but can't see the commit here on GH.
Do you see it as `pull upstream master` ?